Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    rolemodel_rails (0.2.0)
+    rolemodel_rails (0.3.0)
```

Comment:
this seemed to happen automatically when I ran `bundle` locally after cloning the repo.